addons/sourcemod/scripting/SelectiveBhop.sp

```diff
@@ -124,7 +125,7 @@ public void OnClientDisconnect(int client)
 	if(LimitedFlag != LIMITED_NONE)
 	{
 		char sSteamID[64];
-		if(GetClientAuthId(client, AuthId_Engine, sSteamID, sizeof(sSteamID)))
```

Comment:
cvar to control this like other plugins

Reply:
It's not breaking chage since it's only a badckend change and do not have persistant data stored, everything is always cleanned on map change ? (StringMap)

Reply:
oh okay